CHANGES.md

```diff
@@ -82,6 +82,7 @@
 *   Map Preview: Mute errors from users
 *   Map Preview: Report selected distribution to console
 *   Adjusted pagination button styles
+*   Bumped regions index version to 31 because of region shortname
```

Comment:
Only a tiny one: here should be 21.
In the meantime, I will do a test in minikube